src/index.ts

```diff
@@ -152,7 +152,8 @@ const probotHandler = async (robot: Application) => {
   const backportInformationCheck = async (context: Context) => {
     const pr: Octokit.PullsGetResponse = context.payload.pull_request;
 
-    if (pr.base.ref !== 'master') {
+    // TODO(main-migration): Simplify once branch rename is complete.
+    if (pr.base.ref !== 'master' && pr.base.ref !== 'main') {
```

Comment:
There should be a way to do this without hardcoding.

```suggestion
    if (pr.base.ref !== pr.base.repo.default_branch) {
```

Reply:
Other changes like this in GH apps we maintain that non-electron folks use should probably follow a similar approach.

Reply:
Just to clarify, we need to make this change everywhere that references "master" or "main".  Every lookup, check or comparison you changed to check for either.  Should be changed to check for `default_branch` instead.  No more hardcoded strings 🥳

Reply:
yeah, updated throughout.